pkg/api/v1/summary.go

```diff
@@ -54,9 +54,11 @@ func convertSummary(tabSummary *summarypb.DashboardTabSummary) *apipb.TabSummary
 		DetailedStatusMessage: tabSummary.Status,
 		LastRunTimestamp: &timestamp.Timestamp{
 			Seconds: int64(tabSummary.LastRunTimestamp),
+			Nanos:   int32(tabSummary.LastRunTimestamp),
```

Comment:
@ankur12-1610 the idea is to convert the fractional part of the `LastRunTimestamp`. We are currently just rounding it off to fetch seconds but discarding the fractional part.

Reply:
> @ankur12-1610 the idea is to convert the fractional part of the `LastRunTimestamp` coming from `tabSummary` (it is a double there - hence might contain some fractional part of timestamp). We are currently just rounding it off to fetch seconds but discarding the fractional part.
> 
> The fractional part might be useful and needs to be added in nanos to timestamp struct. Once done, the test cases need to be updated too to verify the correctness. Thanks!

Ah! got it!
I've updated the code and used `modf()` for that, I haven't written any tests will do so once the current implemented method is confirmed. Can you please verify?

Reply:
looks good to me @ankur12-1610 ! thanks!